tests/test_go.py

```diff
@@ -79,6 +79,13 @@ def test_build_helm(auto_container_per_test, container_git_clone):
     container with :command:`make` pre-installed.
 
     """
+
+    if (
+        "oldstable" in auto_container_per_test.image_url_or_id
+        or "stable-openssl" in auto_container_per_test.image_url_or_id
+    ):
+        pytest.skip("Helm requires Go >= 1.22")
```

Comment:
Wouldn't it make more sense to explicitly check the go version? Then we won't accidentally skip images in the future in case we forget

Reply:
can do as well